### DISCUSSION TITLE
Wallet Connect v2 does not show Metamask in desktop Link

### DISCUSSION THREAD

**Discussion by rahullilybltr** [WalletConnect/walletconnect-monorepo#3611] [Developer Support] [answered]
Hi,
does anyone know if metamask is supported in desktop  for walletconnect v2?
I am using **@walletconnect/modal** for **WalletConnectV2Adapter**
I am not able to see it in list here.
<img width="356" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/54262706/e6eb6c62-0ce7-4cb4-9a34-e5f131bc7c2a">

**Comment by glitch-txs** [comment, marked as answer]
Hi, this feature is only available on [Web3Modal SDK](https://docs.walletconnect.com/2.0/web3modal/about)
